conans/test/functional/generators/cmake_find_package_test.py

```diff
@@ -13,6 +13,110 @@
 from conans.test.utils.tools import TestClient, NO_SETTINGS_PACKAGE_ID
 
 
+@pytest.mark.slow
+@pytest.mark.tool_cmake
+class TestCMakeFindPackageGenerator:
+
+    @pytest.mark.parametrize("use_components", [False, True])
+    def test_build_modules_alias_target(self, use_components):
```

Comment:
In this test, when `use_components =True`, I can see the following:

**Findhello.cmake**
```cmake
...
set(namespace_BUILD_MODULES_PATHS "/private/var/folders/fc/6mvcrc952dqcjfhl4c7c11ph0000gn/T/tmphcq2mp5yconans/path with spaces/data/hello/1.0/_/_/package/f83037eff23ab3a94190d7f3f7b37a2d6d522241/share/cmake/target-alias.cmake")
...
set(namespace_comp_BUILD_MODULES_PATHS "/private/var/folders/fc/6mvcrc952dqcjfhl4c7c11ph0000gn/T/tmphcq2mp5yconans/path with spaces/data/hello/1.0/_/_/package/f83037eff23ab3a94190d7f3f7b37a2d6d522241/share/cmake/target-alias.cmake")
...
...
foreach(_BUILD_MODULE_PATH ${namespace_BUILD_MODULES_PATHS})
    include(${_BUILD_MODULE_PATH})
endforeach()
```

I would expect the build-module to appear only at the component level... and, of course, a loop calling `include` for each one of them.

Reply:
Still the same, the same build-module appear at component and global level.